Makefile

```diff
@@ -31,6 +31,7 @@ OSTRICH_TARGET = GCC    # can be also MPI but requires mpi compiler; not tested
 UNAME_S := $(shell uname -s)
 DOWNLOAD_CACHE = /tmp/
 FLYINGPIGEON_WPS_URL = http://localhost:8093
+FINCH_WPS_URL = http://localhost:5000
```

Comment:
This assume user know to locally start a Finch and Flyingpigeon server, is that documented somewhere?

If not, maybe point to the Finch and FP on PAVICS by default and user can override on the `make` invocation since these are make variables.

Reply:
Not documented, no. 
That's what happens by default already, no ?

Reply:
It's default if user run on the Jupyter env on Boreas since the default is taken from the notebook. But now in the Makefile, what is set is the "new default" because Makefile explicitly set it.

Reply:
Good point. Could you do it ?

Reply:
Now I understand, I can do it.

Reply:
Check last commit to confirm it's what you meant.